src/screens/CurrencySelectModal.js

```diff
@@ -64,8 +71,19 @@ const TabTransitionAnimation = styled(Animated.View)(
   position.sizeAsObject('100%')
 );
 
+function AndroidWrapper({ children }) {
+  const insets = useSafeArea();
+  const keyboardHeight = useKeyboardArea();
+  const marginBottom = keyboardHeight > 0 ? insets.top + 4 + keyboardHeight : 0;
```

Comment:
Why 4?

Reply:
I'd just move the 4 to a const with the name that represent it better (like androidPadding) or whatever

Reply:
4 is basically little padding that iOS has, without it, the bottom part will be at the same height as the keyboard. will move it to the constant.